pom.xml

```diff
@@ -163,6 +164,12 @@
         <version>${rxjava.version}</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>com.kohlschutter.junixsocket</groupId>
+        <artifactId>junixsocket-core</artifactId>
+        <version>${junixsocket.version}</version>
+        <type>pom</type>
```

Comment:
So this will pull all deps? Is this not optional? Not everyone needs UDS support.

Reply:
You are right, my bad.